cache/cache.cpp

```diff
@@ -671,3 +676,34 @@ int dnet_cmd_cache_io(struct dnet_backend *backend,
 	}
 
 }
+
+int n2_cmd_cache_io(struct dnet_backend *backend,
+                    struct dnet_net_state *st,
+                    struct n2_request_info *req_info,
+                    struct dnet_cmd_stats *cmd_stats,
+                    struct dnet_access_context *context) {
+	auto &cmd = req_info->request->cmd;
+
+	if (cmd.flags & DNET_FLAGS_NOCACHE)
+		return -ENOTSUP;
+
+	auto cache = backend->cache();
+	if (!cache)
+		return -ENOTSUP;
```

Comment:
I didn't support it because it was logged under condition `if (io && (io->flags & DNET_IO_FLAGS_CACHE))` -- only for old (not `_NEW`) commands.

Ok, I'll log.

Reply:
Where I must get io_flags? I have only cflags.

Reply:
I inserted log without any conditions. It can be some spam.

Reply:
It could be obtained from the request.

Reply:
`io_flags` aren't common, are they? Many requests haven't `dnet_io_attr`.

Reply:
They aren't, but they are presented in most of io requests like read, write and remove.